packages/hop-node/src/watchers/classes/ContractBase.ts

```diff
@@ -135,7 +135,15 @@ export default class ContractBase extends EventEmitter {
   }
 
   get waitConfirmations () {
-    return globalConfig.networks[this.chainSlug]?.waitConfirmations ?? 0
+    const chainConfig = globalConfig.networks[this.chainSlug]
+    if (!chainConfig) {
+      throw new Error(`config for chain ${this.chainSlug} not found`)
+    }
+    const { waitConfirmations } = chainConfig
+    if (typeof waitConfirmations !== 'number') {
```

Comment:
See comment above about checking for the value `0` here.

Reply:
Resolved with https://github.com/hop-protocol/hop/pull/346/commits/36a93478a3aa1d9fccd0c560ea29b4a32c1059bb